app/services/workflow_state.py

```
@@ -276,42 +278,43 @@ async def create_savepoint(
+                checksum=self._calculate_checksum(context.state)
+            )
 
-        # Increment sequence
-        self._sequence_counters[workflow_id] = self._sequence_counters.get(workflow_id, 0) + 1
-        sequence = self._sequence_counters[workflow_id]
+            # Store in database
+            try:
+                self.supabase.table("workflow_savepoints").insert(
+                    savepoint.to_dict()
+                ).execute()
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)